packages/next-adapter/package.json

```diff
@@ -60,7 +60,6 @@
   "dependencies": {
     "@expo/package-manager": "0.0.33",
     "@expo/webpack-config": "0.12.43",
-    "@types/next": "^8.0.6",
```

Comment:
We export next.js types from this package

Reply:
it looks like in next@9 this becomes unnecessary https://www.npmjs.com/package/@types/next

Reply:
It is unnecessary, hence why I said it's deprecated, not to mention it isn't used anywhere else in the library.
Although thank you for being critical rather than just blindly merging, that's always good to see.